json/midrise.json

```diff
@@ -0,0 +1,68 @@
+{
+  "schema": "https://github.com/rd2/tbd/blob/master/tbd.schema.json",
+  "description": "sorting out multiple story PSI sets (midrise)",
+  "psis": [
+    {
+      "id": "compliant",
+      "rimjoist": 0.3,
+      "parapet": 0.325,
+      "fenestration": 0.35,
+      "corner": 0.45,
```

Comment:
If a model has a concave corner but the PSI set only has corner, will that edge fall back to use corner?

Reply:
Yup.